persist/src/main/resources/db/migration/V4__Verification_Keys.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS verification_keys (
```

Comment:
Please stay consistent with using all caps or lowercase

Reply:
convention is to use all caps for sql language and lowercase for table names, fields, etc. unless otherwise desired. (or at least that's how I was taught in database design)

[Further explanation](http://www.dpriver.com/blog/2011/09/29/case-capitalization-in-sql-statement/)